test/unit/calculators/vat_payment_deadlines_test.rb

```diff
@@ -7,50 +7,57 @@ class VatPaymentDeadlinesTest < ActiveSupport::TestCase
         .to_return(body: File.open(fixture_file("bank_holidays.json")))
     end
 
+    def new_calculator(date, payment_method)
+      calculator = VatPaymentDeadlines.new
+      calculator.period_end_date = Date.parse(date)
+      calculator.payment_method = payment_method
+      calculator
+    end
+
     context "payment dates for direct-debit" do
       should "calculate last_payment_date where end of month is not a work day" do
-        %w[direct-debit].each do |pay_method|
-          calc = VatPaymentDeadlines.new(Date.parse("2013-10-31"), pay_method)
-          assert_equal Date.parse("2013-12-04"), calc.last_payment_date
+        payment_method = "direct-debit"
 
-          calc = VatPaymentDeadlines.new(Date.parse("2014-04-30"), pay_method)
-          assert_equal Date.parse("2014-06-04"), calc.last_payment_date
```

Comment:
How come we've change the dates in this test?

Reply:
We haven't - it's an artefact of the way git is seeing the file change and aligning them:

Old test    New test
13-14 -->  21-22
16-17 -->  24-25
19-21 -->  27-28